### DISCUSSION TITLE
sphinx-tabs vs sphinx-inline-tabs vs sphinx-design

### DISCUSSION THREAD

**Discussion by astrojuanlu** [executablebooks/meta#955] [General] [answered]
Any thoughts on sphinx-tabs vs sphinx-inline-tabs vs sphinx-design? Evaluating which one to use for Kedro https://github.com/kedro-org/kedro/issues/2366

**Comment by chrisjsewell** [comment, marked as answer]
I would say if you only want tabs then maybe `sphinx-inline-tabs`;  that and `sphinx-design` basically share exactly the same design and stylings for the tabs (last I looked). But then obviously if you want/need any of the other sphinx-design features, then use that.
Personally, I'm not really a fan of the sphinx-tabs style, and also they use JavaScript, whereas the other two are purely CSS based
